src/inspect_ai/_view/www/src/app/log-list/LogsPanel.tsx

```diff
@@ -92,13 +92,6 @@ export const LogsPanel: FC<LogsPanelProps> = ({ maybeShowSingleLog }) => {
     }
   }, [watchedLogs]);
 
-  // Focus the panel when it loads
-  useEffect(() => {
-    setTimeout(() => {
-      rootRef.current?.focus();
-    }, 10);
-  }, []);
```

Comment:
This was preventing keyboard navigation from working without fist clicking on the grid. Thoughts?

Reply:
I think this is fine - it was just focusing the control on load, and the changes in LogListGrid accomplish the same thing.